tests/components/bouncie/test_config_flow.py

```diff
@@ -0,0 +1,143 @@
+"""Test the bouncie config flow."""
+from unittest.mock import patch
+
+from homeassistant import config_entries
+from homeassistant.components.bouncie.config_flow import CannotConnect, InvalidAuth
+from homeassistant.components.bouncie.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+
+async def test_form(hass: HomeAssistant) -> None:
+    """Test we get the form."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == FlowResultType.FORM
+    assert result["errors"] is None
+
+    with patch(
+        "bounciepy.async_rest_api_client.AsyncRESTAPIClient.get_access_token",
+        return_value=True,
+    ), patch(
+        "homeassistant.components.bouncie.async_setup_entry",
+        return_value=True,
+    ) as mock_setup_entry:
+        result2 = await hass.config_entries.flow.async_configure(
+            result["flow_id"],
+            {
+                "client_id": "mock_client_id",
+                "client_secret": "mock_client_secret",
+                "code": "mock_authorization_code",
+                "redirect_uri": "http://mock-redirect-url",
```

Comment:
You can use `from homeassistant.const import CONF_CLIENT_ID, CONF_CLIENT_SECRET` + in your integration const file.

On all the file.